File: Daily Pratice Question/C LANGUAGE/Question(17).c
// 17. Count how many positive and negative numbers are in
//  arr = [1, -2, 3, -4, 5, -6] and print the result.

#include<stdio.h>
  int main(){

      int arr[] = {-3,-5,-2,-5,-4,-9};
      int i,flag;
    int len = sizeof(arr)/sizeof(arr[0]);
    for(i=0;i<len;i++){
        if(arr[i]>0){
            flag++;
        }
    }
    printf("Positve Number in array is %d\n",flag);
flag=len-flag;
printf("Negative Number in array is %d\n",flag);
return 0;
 }

File: Daily Pratice Question/JAVASCRIPT/question(17).js
// 17. Count how many positive and negative numbers are in 
// arr = [1, -2, 3, -4, 5, -6] and print the result.


let arr = [1, -2, 3, -4, 5, -6]
let flag=0
for(i=0;i<arr.length;i++){
    if(arr[i]>0){
        flag++
    }
}console.log("positive number are",flag)
flag=arr.length-flag
console.log("NEgative number are",flag)